src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingdeprecated/Example1.txt

```diff
@@ -1,30 +0,0 @@
-/*xml
-<module name="Checker">
-  <module name="TreeWalker">
-    <module name="MissingDeprecated"/>
-  </module>
-</module>
-*/
-
-// xdoc section -- start
-@Deprecated
-public static final int MY_CONST = 13; // ok
-
-/** This javadoc is missing deprecated tag. */
-@Deprecated
-public static final int COUNTER = 10; // violation
-
-/**
- * @deprecated
- * <p></p>
- */
-@Deprecated
-public static final int NUM = 123456; // ok
-
-/**
- * @deprecated
```

Comment:
There was a violation here. And, the violation was `// violation, tight HTML rules not obeyed`. And, i was confused if it made any sense have to this violation here. I tried to get MSG_UNCLOSED_HTML_TAG but i couldn't the tests stating `java.util.MissingResourceException`